modules/scale-operator-and-deploymentconfigs.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: PROCEDURE
+
+//included in backing-up-and-restoring-3scale-using-oadp.adoc assembly
+
+[id="scale-operator-and-deploymentconfigs_{context}"]
+= Scale operator and deploymentConfigs
+
+Scale up the 3scale operator and any deploymentConfigs that were manually scaled down. After a few minutes, 3scale installation should be fully functional, and its state should match the backed-up state.
+
+.Prerequisites
+
+* Ensure that there are no DeploymentConfigs (DCs) scaled up already, or any extra pods are running. 
+There might be some system-mysql or backend-redis pods running detached from DC after restoration, which can be removed after the restoration is successful.
+
+
+.Procedure
+
+. Scale operator and DC:
++
+[source,terminal]
+----
+$ oc scale deployment threescale-operator-controller-manager-v2 --replicas=1 -n threescale
+----
++
+[source,terminal]
+----
+$ cat ./scaledc.sh
+
+for dc in apicast-production apicast-staging backend-cron backend-listener backend-redis backend-worker system-app system-memcache system-mysql system-redis system-searchd system-sidekiq zync zync-database zync-que; do
+    oc scale dc/$dc --replicas=1 -n threescale
+Done
+----
+
+. Get the `3scale-admin` route to login to the 3scale UI:
++
+[source,terminal]
+----
+$ oc get routes -n threescale
+
+NAME                         HOST/PORT                                                                   PATH   SERVICES             PORT      TERMINATION     WILDCARD
+backend                      backend-3scale.apps.custom-cluster-name.openshift.com                         backend-listener     http      edge/Allow      None
+zync-3scale-api-b4l4d        api-3scale-apicast-production.apps.custom-cluster-name.openshift.com          apicast-production   gateway   edge/Redirect   None
+zync-3scale-api-b6sns        api-3scale-apicast-staging.apps.custom-cluster-name.openshift.com             apicast-staging      gateway   edge/Redirect   None
+zync-3scale-master-7sc4j     master.apps.custom-cluster-name.openshift.com                                 system-master        http      edge/Redirect   None
+zync-3scale-provider-7r2nm   3scale-admin.apps.custom-cluster-name.openshift.com                           system-provider      http      edge/Redirect   None
+zync-3scale-provider-mjxlb   3scale.apps.custom-cluster-name.openshift.com                                 system-developer     http      edge/Redirect   None
+----
+Use the URL from this output to log in to the 3scale admin.
+
+
+.Verification
+
+* Verify if the existing data before creating the back up is available
```

Comment:
```suggestion
* Verify that the existing data is available before trying to create a backup.
```
How? This is not a verification step if there's no actual instruction; this would just be an IMPORTANT admonition instead.